### DISCUSSION TITLE
Recreational Fleet Structure - Length comps

### DISCUSSION THREAD

**Discussion by aliwhitman** [shcaba/Sebastes_melanops_OR#11] [General]
Length compositions confirm differences by shore/estuary modes versus ocean boat fish (from available MRFSS biological data).  

![black_density plot length by mode_MRFSS](https://user-images.githubusercontent.com/80843508/214383571-22db40a9-6169-4cb5-9641-a3c483c2c48e.jpg)

Length comps from RecFIN confirm that we don't need to separate charter and private boat modes within the ocean boat fleet.  

![black_density plot length by mode](https://user-images.githubusercontent.com/80843508/214383970-9f813f94-3230-467c-ae04-1c8dbbd78405.jpg)

**Comment by shcaba** [comment]
These are terrific. I will get these into the pre-assessment discussion. Have you dumped these anywhere in the GitHub repository or in a Google Folder, or should I just grab then from here?

**Comment by aliwhitman** [comment]
I can put them in google along with the underlying data (today!).

**Comment by shcaba** [comment]
I have added these into the pre-assessment presentation.
